### ISSUE TITLE
False positive about `validateAttributeSeperator` with attributes starting with `:` or `@`

### ISSUE THREAD

**Issue by iwata** [sourceboat/pug-lint-vue#13]
I'm using v0.3.0.

About this template example,

```vue
<template>
b-textarea(
  rows='20'
  :value='body'
  :loading='isProcessing'
  @input='inputBody'
  required
)
</template>
```

and set this `.pug-lintrc.js`,

```javascript
module.exports = {
  validateAttributeSeparator: {
    separator: ', ',
    multiLineSeparator: '\n'
  }
}
```

This case reports this:
```
  4:3    Invalid attribute separator found
  5:3    Invalid attribute separator found
  6:3    Invalid attribute separator found
```

It means that `:` and `@` occurred `Invalid attribute separator found` errors.

**Comment by pehbehbeh** [comment]
Does this happen with [pug-lint](https://github.com/pugjs/pug-lint) itself?

**Comment by iwata** [comment]
@pehbehbeh 
No, `pug-lint-vue` about SFC `.vue` file.
I updated the above code to make it easier to understand as examples.

**Comment by SebinSong** [comment]
Hi, I'm currently having the same problem with attributes starting with :. Since, it's a valid syntax from Vue's perspective, I think pug-lint-vue has to consider it valid as well :). Thanks!

**Comment by pehbehbeh** [comment]
This is not an issue of `pug-lint-vue`, but rather an issue of the `pug-lint`. We cannot fix this because we are simply using `pug-lint` under the hood. The issue with the Code above is the indentation.

The following `.pug-lintrc.js` will work (notice the spaces in `multiLineSeparator`):

```js
module.exports = {
    validateAttributeSeparator: {
        separator: ',',
        multiLineSeparator: '\n  '
    }
}
```

But you will get problems with multi-level-indentation.

There is also an ongoing issue here: https://github.com/pugjs/pug-lint/issues/157

**Comment by pehbehbeh** [comment]
Also note that `:` and `@` are not the issue here. The following will get you exactly the same errors:

```
b-textarea(
  rows='20'
  v-bind:value='body'
  v-bind:loading='isProcessing'
  v-on:input='inputBody',
  required
)
```
